src/main/java/org/kohsuke/github/GHExternalGroup.java

```diff
@@ -0,0 +1,263 @@
+package org.kohsuke.github;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * An external group available in a GitHub organization.
+ *
+ * @author Miguel Esteban Gutiérrez
+ */
+public class GHExternalGroup extends GitHubInteractiveObject implements Refreshable {
+
+    /**
+     * A reference of a team linked to an external group
+     *
+     * @author Miguel Esteban Gutiérrez
+     */
+    public static class GHLinkedTeam {
+
+        /**
+         * The identifier of the team
+         */
+        @JsonProperty("team_id")
+        @SuppressFBWarnings(value = "UWF_UNWRITTEN_FIELD", justification = "Field comes from JSON deserialization")
+        private long id;
```

Comment:
```suggestion
        @SuppressFBWarnings(value = "UWF_UNWRITTEN_FIELD", justification = "Field comes from JSON deserialization")
        private long teamId;
```

Reply:
Do this same thing for other fields. Jackson will interpret.  I'm fine with the method names remaining, but would rather not add unneeded annotations.

Reply:
Done.